Squirrely-Ducks/bangazon-sprint-3
# Bm add payment type

## DESCRIPTION
What did you do? 
Why did you do it?
added routes, controllers for adding payment types and displaying when going to the route
## STEPS TO TEST
1. npm start
2. login go to my account s
3. select add payment type
4.enter information, you should then be redirected to see your payment types

## REFERENCE
Does it reference a ticket?
#21
